lib/compat.php

```diff
@@ -228,6 +228,25 @@ function gutenberg_get_post_type_capabilities( $user, $name, $request ) {
 	return $value;
 }
 
+/**
+ * Make sure oEmbed REST Requests apply the WP Embed security mechanism for WordPress embeds.
```

Comment:
This comment is a bit light on rationale for why the security mechanism should be applied. Might be worth elaborating or adding a link to context.

Reply:
In 6dd9e9a i've added a link to the relative [Core ticket](https://core.trac.wordpress.org/ticket/32522)